.cirrus.yml

```diff
@@ -84,10 +88,7 @@ task:
 
 task:
   name: "i686: Linux (Debian stable)"
-  container:
-    dockerfile: ci/linux-debian.Dockerfile
-    cpu: 1
-    memory: 1G
+  << : *LINUX_CONTAINER
```

Comment:
Is this a yaml trick?

Reply:
Yes, we use this in other places, too.

Reply:
Found it: https://yaml.org/type/merge.html
This is interesting, I did not know yaml has this sort of things

Reply:
Yeah, core uses it all over the place, so we're in good company.